internal/savedjobs/repository.go

```diff
@@ -0,0 +1,29 @@
+package savedJobs
+
+import (
+	"database/sql"
+	"time"
+)
+
+type Repository struct {
+	db *sql.DB
+}
+
+func NewRepository(db *sql.DB) *Repository {
+	return &Repository{db}
+}
+
+func (r *Repository) SaveJob(jobID int, developerID string) error {
+	_, err := r.db.Exec(
+		`INSERT INTO developer_jobs_bookmark (job_id, developer_id, saved_at) VALUES ($1, $2, $3)`,
```

Comment:
can you add this table into the schema migration at the root folder? and a couple of fixtures also please? thx